tests/cases/resource_test.py

```diff
@@ -455,10 +455,26 @@ def testGetResourceByPath(self):
                 str(resp.json['_id']), str(item['_id']))
 
         # test bogus path
+        # test is not set
         resp = self.request(path='/resource/lookup',
                             method='GET', user=self.user,
-                            params={'path': '/bogus/path'})
+                            params={'path': '/collection/bogus/path'})
```

Comment:
The two requests below test the case of invalid collection paths, this one was testing [this branch](https://github.com/girder/girder/blob/master/girder/api/v1/resource.py#L204). I think just changing it back to `/bogus/path` covers all of our bases.